frontend/src/e2e-playwright/pages/employee/vasu/vasu.ts

```diff
@@ -2,21 +2,136 @@
 //
 // SPDX-License-Identifier: LGPL-2.1-or-later
 
-import { Page, TextInput } from '../../../utils/page'
+import { Page, TextInput, Element } from '../../../utils/page'
 
-export default class VasuPage {
-  constructor(private readonly page: Page) {}
+export class AuthorsSection extends Element {
+  constructor(element: Element) {
+    super(element)
+  }
+
+  readonly primaryInputSelector = '[data-qa="multi-field-question"] input'
+  readonly #primaryFirstNameInput = this.findAll(
+    this.primaryInputSelector
+  ).first()
+  readonly #primaryLastNameInput = this.findAll(this.primaryInputSelector).nth(
+    1
+  )
+  readonly #primaryTitleInput = this.findAll(this.primaryInputSelector).nth(2)
+  readonly #primaryPhoneNumberInput = this.findAll(
+    this.primaryInputSelector
+  ).nth(3)
```

Comment:
Perhaps something like this would be less boilerplate:
```
#primaryInputs = this.findAll('[data-qa="multi-field-question"] input')
primaryFirstNameInput = new TextInput(this.#primaryInputs.nth(0))
primaryLastNameInput = new TextInput(this.#primaryInputs.nth(1))
primaryTitleInput = new TextInput(this.#primaryInputs.nth(2))
primaryPhoneNumberInput = new TextInput(this.#primaryInputs.nth(3))